packages/twig/src/patterns/components/hero/hero.twig

```diff
@@ -1,15 +1,20 @@
 {#
   HERO COMPONENT
 #}
-
 <div class="{{prefix}}--hero {{prefix}}--hero--{{theme}} {{prefix}}--hero--{{types}} {% if image %}{{prefix}}--hero--image{% endif %}">
-  {% include "@components/image/image.twig" with {
-    alt: image.alt,
-    caption: image.caption,
-    credit: image.credit,
-    prefix: prefix,
-    url: image.url,
-  } only %}
+
+  {% if not image %}
+    <figure class="{{prefix}}--bg" data-prefix="{{prefix}}">
```

Comment:
Can you have this be a div when it's not holding an image? That way it will be semantically neutral.

Reply:
Even better!